annotation/app/annotations/main.py

```diff
@@ -601,7 +628,11 @@ def load_page(
             f"{page_revision['file_id']}/{page_revision['page_id']}"
             ".json"
         )
-        page_obj = s3_resource.Object(tenant, page_path)
+        if S3_PREFIX:
+            bucket_name = f"{S3_PREFIX}-{tenant}"
+        else:
+            bucket_name = tenant
```

Comment:
Let's move it to the new function so we won't mix it up here.
Example from `users`
```python
def get_bucket_name(tenant: str) -> str:
    return f"{config.S3_PREFIX}-{tenant}" if config.S3_PREFIX else tenant
```

Reply:
agreed